Library/PhotosCollectionLayout.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+class PhotosCollectionLayout: UICollectionViewFlowLayout {
```

Comment:
Not sure if splitting in two would help since there are a few details that are different, at the core this two layouts are similar.